src/selectedtests/test_mappings/create_test_mappings.py

```diff
@@ -1,5 +1,6 @@
 """Test Mappings class to create test mappings."""
 import os.path
+import re
```

Comment:
You had fun with naming conventions for all these classes!

Reply:
What do you mean?

Reply:
Starting non pytest files with the prefix 'test_' in python is a bit of a pain.

Reply:
Ah, yes